mysql-test/pom.xml

```diff
@@ -5,9 +5,9 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.opendatakit</groupId>
+        <groupId>org.odk-x</groupId>
```

Comment:
We have not changed the namespace yet, this will cause problems. Again once we have a build system we will do a complete change.